Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8
+FROM python:3.8-slim-buster
```

Comment:
Hi @francrodriguez, just curious to hear your thoughts on this line in the image documentation? Do you have any concern with using a larger image for development?

> python:<version>-slim
>
>This image does not contain the common packages contained in the default tag and only contains the minimal packages needed to run python. Unless you are working in an environment where only the python image will be deployed and you have space constraints, we highly recommend using the default image of this repository.

Reply:
For reference https://hub.docker.com/_/python under the **Image Variants** section

Reply:
Is there anything that we need from the base image vs the `slim-buster`?

Normally it's a good approach to keep everything as lean as possible. Remember that this will be deployed to a Docker registry, where each accepted change (PR) will be deployed as a new tagged image.

In the CD pipeline will also "auto" deploy it into our k8s cluster, keeping the last N (ie 10) versions ready to be quickly swapped.

It's a huge amount of data that we can try to free.

What do you think @mcastellin?

Reply:
I agree, if we can use a lighter image and everything works, much better. WIll be fast downloading and uploading and also in the deploys.